### DISCUSSION TITLE
[Project beta] Hide columns, when filtering for status

### DISCUSSION THREAD

**Discussion by BirgitPohl** [community/community#12342] [Projects and Issues] [answered]
When the filter is active for status, on the project board, I don't want to see the column.
<img width="1007" alt="Screenshot 2022-03-03 at 14 08 48" src="https://user-images.githubusercontent.com/4340405/156570957-061fa41e-e88d-45fa-a706-054004f2cde6.png">

Since the items of the column are hidden anyway, we can also hide the column itself.
This way, we can define a status road, that does not belong to a specific view.
What I want to achieve is that I can define some backlog status, that shall not be visible on the development board.

**Comment by StoneCypher** [comment, marked as answer]
See #6906
